src/components/API.js

```diff
@@ -8,131 +8,113 @@
     console.log(result);
   });*/
 
-export default class Api {
-  constructor(name, link) {
-    this.name = name;
-    this.link = link;
-  }
-  getAppInfo() {
-    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
-  }
-
-  getInitialCards() {
-    return this._request("https://around.nomoreparties.co/v1/group-12/cards", {
-      method: "GET",
-      headers: {
+  export default class Api {
+    constructor(name, link) {
+      this.name = name;
+      this.link = link;
+      this._baseURL = "https://around.nomoreparties.co/v1/group-12";
+      this._headers = {
```

Comment:
Please, don't hardcode the baseUrl inside the class. You pass all data into the constructor


```js
export default class Api {
  constructor({ baseUrl, headers }) {
    this._baseURL = baseUrl;
    this._headers = headers;
  }

```

Reply:
done